web/thesauruses/python/3/queues_stacks.json

```diff
@@ -0,0 +1,12 @@
+{
+    "meta": {
+      "language": "python",
+      "language_version": "3.10",
+      "language_name": "Python",
+      "structure": "Queue And Stacks"
+    },
+    "concepts": {
+      "queue": {
+        "code": "def function_name():\n    statements",
+        "name": "Function that does not return a value and takes no parameters"
+      },
```

Comment:
The number of `{`s don't match the number of `}`s

Reply:
I don't know how this happened. This is not what i wrote, this is just the start of all that which i copied over as starter code. Let me check it and i will have it resolved as soon as possible.

I am sure what i wrote was proper because i always double check it and i remember checking all the brackets.